### PR TITLE
Update `poly_gate` to init lookup wire with zero

### PR DESCRIPTION
**Pull request by vlopes11**

### CODE REVIEW

File: src/constraint_system/composer.rs [hunk 1]
```diff
@@ -322,7 +322,7 @@ impl TurboComposer {
         self.q_logic.push(BlsScalar::zero());
         self.q_fixed_group_add.push(BlsScalar::zero());
         self.q_variable_group_add.push(BlsScalar::zero());
-        self.q_lookup.push(BlsScalar::one());
+        self.q_lookup.push(BlsScalar::zero());
```
Comment (CPerezz) [inline src/constraint_system/composer.rs:325]: That change makes sense to me. Was going to report in in the review `add_lookups` branch

File: src/plookup/multiset.rs [hunk 2]
```diff
@@ -37,11 +37,13 @@ impl MultiSet {
 
     /// Generate a `MultiSet` struct from a slice of bytes.
     pub fn from_slice(bytes: &[u8]) -> Result<MultiSet, Error> {
-        let mut buffer = bytes;
+        let buffer = bytes;
```
Comment (ureeves) [inline src/plookup/multiset.rs:40]: Why not remove `buffer` entirely?

Comment (vlopes11) [reply src/plookup/multiset.rs]: Good catch, fixed

**Review by CPerezz — APPROVED** [review]
LGTM. Merge ASAP into @Bounce23 's branch :)

**Pull request merged**